djangoproject/templates/accessibility/index.html

```diff
@@ -0,0 +1,150 @@
+{% extends "accessibility/base.html" %}
+{% load i18n %}
+
+{% block title %}{% translate "Django Accessibility Statement" %}{% endblock %}
+
+{% block content %}
+
+  <h1>{% translate "Django Accessibility Statement" %}</h1>
+
+  <p>
+    {% blocktranslate trimmed %}
+      The Django Software Foundation is committed to fostering an inclusive and
+      accessible community, project, and ecosystem. Accessibility is integral to
+      creating a space where everyone — regardless of ability — can contribute,
+      build, and benefit from the Django framework and its broader community.
+    {% endblocktranslate %}
+  </p>
+
+  <h2>{% translate "Our commitment" %}</h2>
+
+  <p>
+    {% blocktranslate trimmed %}
+      We strive to make Django itself and its related online and in-person spaces
+      accessible for all. This includes:
+    {% endblocktranslate %}
+  </p>
+
+  <ul class="simple">
```

Comment:
```suggestion
  <ul>
```

doesn’t seem to do anything / be defined anywhere?